### ISSUE TITLE
Question: How to change the padding between lines in a code block?

### ISSUE THREAD

**Issue by 6a6f6a6f** [gpoore/minted#429]
## Summary

I'm using `minted` with Pandoc with a template made by the company I work. To set up the package, I had it done as follows:

```tex
\renewcommand\theFancyVerbLine{%
  \textcolor{DraculaBackground}{\tiny\arabic{FancyVerbLine}}%
}

\setminted{
	style=dracula,
	bgcolor=DraculaBackground,
	breaklines=true,
    mathescape=true,
	autogobble=true,
	linenos=true,
	frame=none,
	fontsize=\small,
	formatcom=\color{DraculaForeground}
}
```

Since the default background of the template is `white` I have to patch the color of the line numbers to be visible. But when I compiled it using `pdflatex` I had the following output:

![Generated code block](https://github.com/user-attachments/assets/9aba3e1f-5af6-4492-a891-cad4e60632b2)

My goal is to remove the padding to remove the white lines from the original document background.

## Environment

- `pandoc`: v3.6 (with Lua v5.4 as the scripting engine)
- `latexminted`: v0.3.1 (`latex2pydata` v0.4.0, `latexrestricted` v0.6.0 and `pygments` v2.18.0)
- `pdfTeX`: v3.141592653-2.6-1.40.26

I'm using the [`pandoc/lua-filters`](https://github.com/pandoc/lua-filters) from [4cbb8d8](https://github.com/pandoc/lua-filters/commit/4cbb8d8e93958a998a49e42d0b49f19014761030) with the filter [`minted.lua`](https://github.com/pandoc/lua-filters/blob/4cbb8d8e93958a998a49e42d0b49f19014761030/minted/minted.lua).

**Comment by muzimuzhi** [comment]
According to package manual of `minted` v3,

> - `bgcolor` now uses the new `bgcolor` option from `fvextra` v1.8. Because `bgcolor`
now introduces no additional whitespace or padding, existing documents may
require some modification. Added new option `bgcolorpadding` for modifying
padding in background color regions. Added new option `bgcolorvphantom` for
setting height of background color in inline contexts. When more sophisticated
background colors are needed, `tcolorbox` or a similar package should be used.

Could you provide a complete LaTeX example?

**Comment by gpoore** [comment]
I believe that the white lines are an artifact of the PDF viewer. When I reimplemented `bgcolor`, I didn't see the lines at all in several browsers and Acrobat Reader, but did see them in Sumatra PDF. Depending on which PDF viewer you are using, the lines may not exist.

If they do exist in the viewer that you want to use, then it may be simpler to switch to using `tcolorbox` instead...it seems more robust against artifacts. Otherwise, there might be a way to adjust the line spacing to cause some overlap in the background color...if these show up in enough cases, I may need to modify the implementation.

**Comment by gpoore** [comment]
I've just released `fvextra` v1.11.0, which `minted` uses internally to create background colors. This modifies the way that background colors are created so that PDF viewer artifacts are eliminated for most software that has this issue.

**Comment by solonovamax** [comment]
> I've just released `fvextra` v1.11.0, which `minted` uses internally to create background colors. This modifies the way that background colors are created so that PDF viewer artifacts are eliminated for most software that has this issue.

I'm still able to see these white lines when using Evince. It is much easier to see the lines when zoomed out.
I can also see it in chromium's pdf viewer, when using the `bgcolorpadding` option.

I'm currently using v1.12.0 of `fvextra`

I've found that if I do `\fvset{bgcolorboxoverlap=1pt}`, it seems to remove the white lines that appear between each line, however I cannot seem to get the white lines from the border to disappear.
